actionpack/lib/action_dispatch/journey/nodes/node.rb

```diff
@@ -65,7 +65,7 @@ def initialize(x = Object.new)
         def literal?; false; end
       end
 
-      %w{ Symbol Slash Dot }.each do |t|
+      %w{ Slash Dot }.each do |t|
```

Comment:
How about expanding it for all classes?

Originally we had two class evals for `%w{ Symbol Slash Literal Dot }` and `%w{ Cat Group Star Or }`.
https://github.com/rails/journey/blob/0dfe8ba78e9ab31a985412c2275781a0c0376553/lib/journey/nodes/node.rb

But now one class eval for `%w{ Cat Group Star Or }` is already expanded at https://github.com/rails/journey/commit/bc37b94050c09f13b6bd782d551f59ba492feebc.

The remaining one class eval for `%w{ Slash Dot }` is over meta programming to me.

Reply:
I think it is a great suggestion, I'm definitely applying this! Changes done and squashed.